sklearn/_loss/tests/test_link.py

```diff
@@ -103,6 +103,6 @@ def test_link_out_argument(link):
 
     out = np.empty_like(y_pred)
     raw_prediction_2 = link.link(y_pred, out=out)
-    assert_allclose(raw_prediction, out)
+    assert_allclose(raw_prediction, out, rtol=1e-02)
```

Comment:
Again, this is a very large and therefore loose `rtol`.

Reply:
True. I'm just wondering: is the assertion in line 106 actually needed because `np.empty_like` generates uninitialised random numbers that are close-ish to the input array? If not, the line could be deleted and `global_random_seed` could be used. If it is needed, then I guess not using `global_random_seed` is the better way forward.

Reply:
Line 105 sets `out` kind of implicitly, `link.link(y_pred, out=out)`. This is what is being tested.